CONTRIBUTING.rst

```diff
@@ -67,6 +67,10 @@ Ready to contribute? Here's how to set up `pipreqs` for local development.
 
     $ poetry install --with dev
 
+3.1. Pipreqs is using pre-commits
+
+    $ poetry run pre-commit install
```

Comment:
can' t this be done in the `poetry install --with dev` step somehow?

Reply:
since pre commit is already marked as a dependency, do we need this? or is it installing the hooks and not  package pre-commit?